### PR TITLE
Feature/58 manage aanganwadi

### PR DESCRIPTION
**Pull request by 786ahmedhussain**

### CODE REVIEW

File: Backend/config/database.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 MongoClient()
 
 client = MongoClient(
-    "mongodb+srv://username:password@cluster0.cythueh.mongodb.net/?retryWrites=true&w=majority")
+    "mongodb+srv://root:Emids123@cluster0.jwkiznj.mongodb.net/?retryWrites=true&w=majority")
```
Comment (mjayakumarmca) [inline Backend/config/database.py]: could you please revert this change.

File: FrontEnd/src/views/AddAanganwadi.vue [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  import { useAanganwadiStore } from "../stores/aanganwadi";
+  import router from "../router";
+  let aanganwadi = reactive({
+    aanganwadiName: "",
+    contactPersonName: "",
+    contactPersonPhone: 0,
+    contactPersonEmail:"test@hotmail.com",
+    contactPersonPassword: "",
+    location: "",
+    coordinates: "61.62, 62.61",
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/AddAanganwadi.vue]: remove all these hardcoded default values

File: FrontEnd/src/views/EditAanganwadi.vue [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  import { useAanganwadiStore } from "../stores/aanganwadi";
+  import router from "../router";
+  import axios from "axios";
+  import { useRoute } from "vue-router";
+
+  const route = useRoute();
+
+  let updatedAanganwadi = reactive({
+    id:"",
+    aanganwadiName: "Edit Aanganwadi",
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/EditAanganwadi.vue]: remove all these hardcoded values

File: FrontEnd/src/views/EditAanganwadi.vue [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+  const store = useAanganwadiStore();
+
+  updatedAanganwadi = computed(() => {
+    if(store.currentAanganwadi){
+      return store.currentAanganwadi;
+    }else{
+      return{
+        id: "",
+        aanganwadiName: "Sanath",
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/EditAanganwadi.vue]: Here too remove all hardcoded values

File: FrontEnd/src/views/EditSupplement.vue [hunk 5]
```diff
@@ -0,0 +1,118 @@
+<template>
+    <div class="full-div container" style="width: 1280px">
+      <div id="x-contest" class="container-fluid p-3">
+        <form @submit.prevent="updateNgo">
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/EditSupplement.vue]: can you rename this to updateSupplement

File: FrontEnd/src/views/EditSupplement.vue [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+    }
+  });
+  onMounted(async () => {
+    console.log("Editing.... ");
+    console.log(route.params.id);
+    await store.getSupplement(route.params.id);
+    console.log(store.supplement);
+  });
+  
+  const updateNgo = async () => {
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/EditSupplement.vue]: it should be updateSupplement

File: FrontEnd/src/views/SupplementList.vue [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+                <th scope="col" class="text-center">Actions</th>
+              </tr>
+            </thead>
+            <tbody>
+              <div v-show="!isSupplementsAvailable">
+                <tr>
+                  <td>No records available</td>
+                </tr>
+              </div>
+              <tr v-for="item of ngos" :key="item.id">
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/SupplementList.vue]: change ngos to supplements

File: FrontEnd/src/views/SupplementList.vue [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+  </style>
+  <script setup>
+  import { onMounted, computed } from "vue";
+  import { useSupplementStore } from "../stores/supplement";
+  import swal from "sweetalert";
+  import router from "../router";
+  
+  const store = useSupplementStore();
+  
+  const ngos = computed(() => {
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/SupplementList.vue]: change ngos to supplements

File: FrontEnd/src/views/SupplementList.vue [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+      icon: "warning",
+      buttons: true,
+      dangerMode: true,
+    }).then((willDelete) => {
+      if (willDelete) {
+        store.deleteSupplement(id);
+        swal("Supplement has been deleted!", {
+          icon: "success",
+        });
+        ngos.value = store.fetchSupplements();
```
Comment (mjayakumarmca) [inline FrontEnd/src/views/SupplementList.vue]: change ngos to supplements

**Review by mjayakumarmca — CHANGES_REQUESTED** [review]
Given some comments to address

File: FrontEnd/src/views/EditSupplement.vue (hunk 6 above)
Comment (786ahmedhussain) [reply FrontEnd/src/views/EditSupplement.vue]: Done

File: FrontEnd/src/views/EditSupplement.vue (hunk 5 above)
Comment (786ahmedhussain) [reply FrontEnd/src/views/EditSupplement.vue]: Done

File: FrontEnd/src/views/SupplementList.vue (hunk 7 above)
Comment (786ahmedhussain) [reply FrontEnd/src/views/SupplementList.vue]: Done

File: FrontEnd/src/views/SupplementList.vue (hunk 8 above)
Comment (786ahmedhussain) [reply FrontEnd/src/views/SupplementList.vue]: Done

File: FrontEnd/src/views/SupplementList.vue (hunk 9 above)
Comment (786ahmedhussain) [reply FrontEnd/src/views/SupplementList.vue]: Done

File: FrontEnd/src/views/EditAanganwadi.vue (hunk 4 above)
Comment (786ahmedhussain) [reply FrontEnd/src/views/EditAanganwadi.vue]: Done

File: FrontEnd/src/views/EditAanganwadi.vue (hunk 3 above)
Comment (786ahmedhussain) [reply FrontEnd/src/views/EditAanganwadi.vue]: Done

File: FrontEnd/src/views/AddAanganwadi.vue (hunk 2 above)
Comment (786ahmedhussain) [reply FrontEnd/src/views/AddAanganwadi.vue]: Done

**Review by mjayakumarmca — APPROVED** [review]
LGTM
